float32 batterie
int16[] droneAxes
uint16 tofSensor
uint16 thrust
float32[] timing
float32 load
uint16[] spektrumframe
